packages/blocks/src/code-block/code-block.ts

```diff
@@ -167,12 +167,11 @@ export class CodeBlockComponent extends NonShadowLitElement {
     return service.hljs.default.highlight;
   }
 
-  private langUpdated = new Slot<{
-    lang: Lang;
-    highlighter: Highlighter;
-  }>();
+  private _richTextResizeObserver: ResizeObserver = new ResizeObserver(() => {
+    this._updateLineNumbers();
+  });
 
-  private _preLang: Lang | null = null;
+  private _preLang: string | null = null;
```

Comment:
Is this operation safe?

Reply:
I think so. We cannot always assume that this property is legal.